packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -32,6 +31,11 @@ import expandFacet from '@salesforce/label/c.quantic_ExpandFacet';
  * @property {string} [value]
  * @property {number} [index]
  */
+/**
+ * @typedef CaptionProvider
+ * @type {object}
+ * @property {Record<string, string>} captions
+ */
 
 /**
  * A facet is a list of values for a certain field occurring in the results, ordered using a configurable criterion (e.g., number of occurrences).
```

Comment:
Same comment regarding the JS doc comment to  obtain automatic documentation of the slot `captions`.